scripts/fmt.sh

```diff
@@ -13,9 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.#
 
+# Fix: 'fatal: detected dubious ownership in repository',
+# only do this if it is running into the develop image
+if [ "${PWD}" == "/opt/app" ]; then
+  git config --global --add safe.directory "*"
```

Comment:
how about we move this code into the developer image, since it is only needed there

Reply:
Make sense, will take a look on it.